doc/manual/cookbook.rst

```diff
@@ -274,13 +274,13 @@ segmentation.
           e.g. ``conductivities=[0.3]``.
 
 Using this model, the BEM solution can be computed using
-:func:`mne.make_bem_solution`` as::
+:func:`mne.make_bem_solution` as::
 
     >>> bem_sol = make_bem_solution(model)
     >>> write_bem_solution('sample-5120-5120-5120-bem-sol.fif', bem_sol)
 
 After the BEM is set up it is advisable to check that the
-BEM model meshes are correctly positioned using *e.g.*, :class:`mne.Report`.
+BEM model meshes are correctly positioned using *e.g.*, :class:`mne.report.Report`.
```

Comment:
It looks like you've done it already below, so you should revert this line's change